### PR TITLE
Global attribute: role

### LINKED ISSUE #3614: Issue with "HTML attribute reference": missing role attribute
MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes

#### What information was incorrect, unhelpful, or incomplete?

Incomplete: information about the aria [`role`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles) attribute.

Is there a reason that it isn't listed here?

#### What did you expect to see?

A row for the `role` attribute.

#### Did you test this? If so, how?

N/A.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/html/attributes`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/attributes/index.html
* Last commit: https://github.com/mdn/content/commit/1f2db9f0a46b99c254476b5f9445a74b98a6fa30
* Document last modified: 2021-03-27T00:17:29.000Z

</details>

### PR DESCRIPTION
**Pull request by estelle**
Added `role` attribute to the `aria-*` line. It is not defined in the HTML specification but rather in the ARIA specification.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by estelle** [comment]
https://github.com/mdn/content/issues/3614

File: files/en-us/web/html/global_attributes/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Global attributes may be specified on all [HTML elements](/en-US/docs/Web/HTML/E
 In addition to the basic HTML global attributes, the following global attributes also exist:
 
 - {{HTMLAttrDef("xml:lang")}} and {{HTMLAttrDef("xml:base")}} — these are inherited from the XHTML specifications and deprecated, but kept for compatibility purposes.
-- The multiple **[`aria-*`](/en-US/docs/Web/Accessibility/ARIA)** attributes, used for improving accessibility.
+- The ARIA [role](/en-US/docs/Web/Accessibility/ARIA/Roles) attribute and the multiple **[`aria-*`](/en-US/docs/Web/Accessibility/ARIA/Attributes)** states and properties, used for ensuring accessibility.
```
Comment (teoli2003) [inline files/en-us/web/html/global_attributes/index.md:21]:
```suggestion
- The ARIA [`role`](/en-US/docs/Web/Accessibility/ARIA/Roles) attribute and the multiple **[`aria-*`](/en-US/docs/Web/Accessibility/ARIA/Attributes)** states and properties, used for ensuring accessibility.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A nit.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**